cmd/kubeadm/app/apis/output/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +groupName=output.kubeadm.k8s.io
+
+// Package output implements the kubeadm structured output
```

Comment:
[1] It will be great to one or two sentences  about why we are adding this and which are the expected consumers (probably in the KEP there is already something we can copy/paste)

Reply:
Done